timesketch/api/v1/resources/sketch.py

```diff
@@ -361,39 +364,34 @@ def get(self, sketch_id):
                 mapping_dict['type'] = value_dict.get('type', 'n/a')
                 mappings.append(mapping_dict)
 
+        # Get number of events per timeline
         if sketch_indices:
-            # Stats for index. Num docs per shard.
-            for timeline in sketch.active_timelines:
-                index_name = timeline.searchindex.index_name
+            for index_name in sketch_indices:
                 if indices_metadata[index_name].get('is_legacy', False):
                     doc_count, _ = self.datastore.count(indices=index_name)
                     stats_per_timeline[timeline.id] = {'count': doc_count}
```

Comment:
`timeline.id` isn't defined here... that is the `timeline` isn't defined.

Reply:
Yeah, that is what I fixed before removing the draft status :)